openpower/patches/mihawk-patches/linux/Linux-nvme-pci-Fix-nvme-error-detected-kernel-crash.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/drivers/nvme/host/pci.c b/drivers/nvme/host/pci.c
+index 75539f7..2a96431 100644
+--- a/drivers/nvme/host/pci.c
++++ b/drivers/nvme/host/pci.c
```

Comment:
Is this patch submitted upstream?

As per https://open-power.github.io/op-build/process/KernelTree.html we want all Kernel patches to be upstream.

In this case, the patch should also be in `git format-patch` format, so it has a full commit log, authorship details, and preferably where it was cherry-picked from.

Reply:
Hi @mikey , around ~03/2018, Wistron was hitting some petitboot kernel eeh/crash issues only when the nvme disks installed during their reboot regression testing.   At that moment, we got this simple patch from your side and the issue seems to be solved. :)   So, not sure if this patch is still needed or upstreamed already? or we could bypass/ignore this patch and then go ahead to test the "reboot+nvme" with current newer petitboot kernel? thank you.
# Related Boston issue/comments:
https://github.com/open-power/boston-openpower/issues/1076#issuecomment-372496320

Reply:
I just commented in that issue that this patch went upstream in v4.17 as:

> commit f0295e047fcf52ccb42561fb7de6942f5201b676
> Author: Michael Neuling <mikey@neuling.org>
> Date: Mon Mar 26 15:17:07 2018 +1100
> powerpc/eeh: Fix race with driver un/bind

So provided that does fix it, we should just be able to update and get the fix

Reply:
@HankChang -- Can we update to a kernel that includes the fix?